package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.22.19",
   "private": true,
   "scripts": {
-    "dev": "tinacms dev -c \"next dev\"",
+    "dev": "tinacms dev -c \"node --require .\\appInsight-api.js node_modules/next/dist/bin/next dev\"",
```

Comment:
would not have expected that this would be required

Reply:
In order to use appInsight with Next API, it is necessary to load it at compile time. There is no other way to load appInsight and run Next together. This means that appInsight must be loaded while running the Next server.